sources/exception.asciidoc

```diff
@@ -157,29 +157,15 @@ Reasons for raising this exception include:
 * The extracted type is not marked with `@ExtractedValue`
 * The `@ExtractedValue` annotation is given more than once for one value extractor type
 
-[role="api"]
```

Comment:
Ok. Note these are not "examples" in the usual sense, but some of the specific cases where this exception is to be thrown. But I think it's covered in the specific sections, too.